pom.xml

```diff
@@ -91,6 +91,14 @@
         <license.copyrightOwners>EMBL</license.copyrightOwners>
         <enforcer.skip>true</enforcer.skip>
 
+        <!-- Java 8	Update Site	-->
+        <imglib2-cache.version>1.0.0-beta-16</imglib2-cache.version>
+        <bigdataviewer-core.version>10.3.1</bigdataviewer-core.version>
```

Comment:
This is too new!
See my mail.
Let's try 10.2.2

Reply:
Ok